pipelines/loadtest.yml

```diff
@@ -0,0 +1,953 @@
+---
```

Comment:
What's the difference between `concourse/loadtest-pipeline.yml` and `pipelines/loadtest.yml`?